nixos/modules/programs/bash/ls-colors.nix

```
@@ -13,11 +13,17 @@ in
     programs.bash.enableLsColors = lib.mkEnableOption "extra colors in directory listings" // {
       default = true;
     };
+    programs.bash.lsColorsFile = lib.mkOption {
+      type = lib.types.str;
+      default = "";
+      example = lib.literalExpression "\${pkgs.dircolors-solarized}/ansi-dark";
```

you don't need the `\` with `lib.literalExpression`
```suggestion
      example = lib.literalExpression "${pkgs.dircolors-solarized}/ansi-dark";
```

---

Oh, I thought I saw the build because of this

---

Ahh you might be right. I had to search for more example. Sorry ;o;